packages/e2fsprogs.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class E2fsprogs < Package
+  description 'e2fsprogs are ext2/3/4 file system utilities.'
+  homepage 'http://e2fsprogs.sourceforge.net/'
+  version '1.45.6'
```

Comment:
Missing `compatibility 'all'`?

Reply:
I'm not sure if it is all compatible, so I think maybe we should let the package marinate for a week or two and if any arm users report problems we can resolve them before adding compatibility all. Is that okay?

Reply:
The fact that you don't have a line there proves to me that you haven't tested this because it will break crew. :)

Reply:
I have tested it and haven't had a problem on x86_64. Unfortunately i don't own an arm chromebook :(. Has this package been tried before and had issues?